lib/cocoapods/installer.rb

```diff
@@ -200,6 +200,11 @@ def generate_pods_project(generator = create_generator)
     #
     attr_reader :analysis_result
 
+    # @return [Array<Hash{String, TargetInstallationResult}>] the installation results produced by the pods project
+    #         generator
+    #
+    attr_reader :target_installation_results
```

Comment:
unfortunately to land this for validator changes I need the `installer` to hold on to this state. I do not want to fix this now since this is already blowing out of proportion in terms of changes. `installer` needs a clean up in a follow up.